site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/repository/FhirContextWrapper.java

```diff
@@ -18,6 +18,7 @@
 @Component
 public class FhirContextWrapper {
     private final FhirContext fhirContext;
+    private final int RESULT_COUNT = 50;
```

Comment:
Is 50 the right number - might we have more sites/practitioners/whatevers than that?  (Is there any performance penalty if we give a high number here?)

Reply:
Might be worth to externalize as a config setting to control more easily.

Reply:
not sure - just put an arbitrary larger number than 10 - can up to 100 if you want - the amount of data being passed in 1 request rather than multi requests via paging or similar hasnt been clarified etc.. Im sure as the app grows this is something that will be discussed and therefore functionality like this will also change...thinking about it, it maybe best to keep it on the lower side so not to mask any perf issues..if we start getting more than 50 results that might push us into a perf type discussion??

Reply:
fair point bout the config approach - Ill update

Reply:
updated config files and set the default value to 50 - we can easily adjust as per config changes if required - good catch Kate/Tamir..